docs/static/monitoring-apis.asciidoc

```diff
@@ -0,0 +1,312 @@
+[[monitoring]]
+== Monitoring APIs
+
+Logstash provides the following monitoring APIs to retrieve runtime metrics
+about Logstash:
+
+* <<hot-threads-api>>
```

Comment:
@acchen97 Actually, ignore my previous comment. I'll move the section to the end for now and we can decide on the final organization when/if we have more APIs to worry about and the order matters more. :-)